### ISSUE TITLE
New-look Gmail

### ISSUE THREAD

**Issue by myitcv** [kevinwucodes/gmail-quick-links#21]
This is a great plugin! Please can I ask, will you be adding support for the new-look gmail?

Many thanks

**Comment by myitcv** [comment]
Sorry, I've just seen https://github.com/kevinwucodes/gmail-quick-links/commit/0e0b41a7e7edc44dc4e55b60b92465a9cd29da2a.

However when I use the new-look gmail the quick links section appears _above_ the inbox section. Anything required to upgrade/fix this?

**Comment by myitcv** [comment]
<img width="247" alt="screen shot 2018-05-09 at 10 50 45" src="https://user-images.githubusercontent.com/3374574/39808446-de7230a2-5376-11e8-94c1-13e9381f3df2.png">

If it helps, here is a screen shot of what I see

**Comment by kergoth** [comment]
Seeing this issue here too.

**Comment by hefe88** [comment]
Yes, I see this too. I'd like the Quick Links to show below the labels. Can that be fixed? This is a great extension.

**Comment by bennjmin** [comment]
Just FYI

I installed the extension just a few weeks ago, it worked fine for keeping a few complex filters at hand. And they worked fine with the new Gmail layout (I actually appreciated that the links showed up on top).

I didn't visit that email box for a week, but today I noticed the links had disappeared. Even after restarting Chrome or re-activating the extension.

But when I went back to the old Gmail layout, they appeared again (at the bottom). So I currently keep the old layout for this mailbox.

**Comment by yngn** [comment]
I have exactly the same issue as Bennjmin.  I can tell that this behavourial modification is super recent (maybe a couple of days).
I just hope that Google will not force us to move away from the "classical/old" Gmail layout in a near future.

**Comment by bennjmin** [comment]
FYI Today I discovered the issue is solved in the new layout.

**Comment by igordovgaluk** [comment]
displayed above Inbox. 
Is there any patch or workaround to move it below labels?

**Comment by Zanatoo** [comment]
It won't move Quicklinks to the bottom of the column but you could use something like [Stylebot](https://chrome.google.com/webstore/detail/stylebot/oiaejidbmkiecgbjeifoejpgmdaleoha) to allow you use your own custom CSS to style Quicklinks... 

![](https://i.postimg.cc/sDcmsTSb/quicklinks.jpg)

**Comment by jad00gar** [comment]
> It won't move Quicklinks to the bottom of the column but you could use something like [Stylebot](https://chrome.google.com/webstore/detail/stylebot/oiaejidbmkiecgbjeifoejpgmdaleoha) to allow you use your own custom CSS to style Quicklinks...
> 
> ![](https://camo.githubusercontent.com/b13aa270923ad0fc9a22c9f0d9be64b6251e75241f2ad9afbb84eb1596c779e0/68747470733a2f2f692e706f7374696d672e63632f7344636d735453622f717569636b6c696e6b732e6a7067)

would you mind sharing your css
